spring/src/main/java/org/axonframework/spring/config/annotation/SpringHandlerDefinitionBean.java

```diff
@@ -89,6 +89,7 @@ public void afterPropertiesSet() {
      *
      * @param additionalFactories The extra definitions to register
      * @see SpringBeanParameterResolverFactory
+     * @see SpringBeanDependencyResolverFactory
```

Comment:
So, this change did me frown a little, since a `HandlerDefinition` has nothing to do with parameter resolution.
Checked the git history, but this piece of Javadoc was wrongly copied over from the `SpringParameterResolverFactoryBean#setAdditionalFactories(List<ParameterResolverFactory>)` back in the day...
No need to change this in your PR by the way, but adding this line is also unnessary.